docs/getting-started/welcome.md

```diff
@@ -1,36 +1,55 @@
-- [Optimistic oracle service](https://docs.umaproject.org/getting-started/oracle)
-
-Together, these two components enable the creation of fast, efficient, and secure synthetic derivatives on the Ethereum blockchain.
-
-## Understanding UMA
-
-If you would like to learn more about the concepts behind UMA's priceless financial contracts and optimistic oracle, start by reading this [overview](getting-started/overview.md) of the UMA ecosystem.
-
-## For Developers
-
-If you are a developer interested in building with UMA Protocol, view this [explainer](synthetic-tokens/what-are-synthetic-assets.md) of how UMA's priceless synthetic tokens work and to learn more about UMA's expiring synthetic tokens. 
-
-Additionally, the [development guide](build-walkthrough/build-process) to learn how to launch your own expiring synthetic token on UMA. 
-
-You can find additional reference materials, including a list of UMA's testnet and mainnet contracts [here](dev-ref/addresses.md).
-
-## For UMA Tokenholders
-
-If you are interested in the UMA project token, please read this [section](uma-tokenholders/uma-holders.md) to learn more about the role of the UMA project token in the DVM and the rights and responsibilities of UMA project tokenholders.
-
-## Contact 
-If you have any questions or want to connect with the UMA community, please check out our community [page](community/community-overview) or join our [Discord](https://discord.umaproject.org/).
+---
+title: Welcome to UMA
+sidebar_label: Welcome to UMA
+slug: /
+---
+
+UMA stands for "Universal Market Access" 
+
+<!--taken from UMA/TRS Mission and Values -->
+Our mission is to make financial markets universally fair and accessible.
+
+We are driven by a shared belief that financial markets should be universally accessible - unrestrained by censorship, your pre-existing financial and social capital, or your beliefs and values. The economic freedom created by open market access enables everyone equal access and footing to pursue prosperity and build their financial independence.  Free and universal access to financial markets is a prerequisite to this free, open and fair financial system.  
+
+Our goal is to enable anyone to attain or transfer any form of risk seamlessly through the UMA protocol thus empowering everyone to participate in this new universally accessible financial system.
+<!--end-->
+
+UMA builds open source infrastructure which enables developers to create financial instruments in a permissionless, secure and efficient manner.  We work with developers, financial engineers and product designers to support them build products on our infrastructure, while supporting tokenholders to actively participate in the governance of our protocol. 
+
+Specifically this involves 
+ - [Priceess financial contract templates](getting-started/priceless_contracts.md) which can be used to create synthetic tokens
+ - [Governance](getting-started/governance.md) which manages improvements to the UMA protocol

```

Comment:
These links are not resolving correctly and breaking the deployment. Is this because these pages are new pages in other prs?

-`getting-started/priceless_contracts.md`
- `getting-started/governance.md`
- `/priceless-financial-contracts/priceless-contracts.md`